langchain/chains/pal/base.py

```
@@ -55,7 +55,11 @@ def _call(self, inputs: Dict[str, str]) -> Dict[str, str]:
         if self.verbose:
             print_text(code, color="green", end="\n")
         repl = PythonChain()
-        res = repl.run(code + f"\n{self.get_answer_expr}")
+        try:
+            res = repl.run(code + f"\n{self.get_answer_expr}")
+        except:
+            print("Exception!")
+            res = ""
```

i would also return the error, so the agent can see whats going on and adjust